lib/outputs/models.py

```diff
@@ -0,0 +1,170 @@
+from dataclasses import dataclass
+
+from config.config import timestamp
+from lib.check.models import Check_Report, Organizations_Info
+
+
+@dataclass
+class Compliance_Framework:
+    Framework: str
+    Version: str
+    Group: list
+    Control: list
+
+
+@dataclass
+class Check_Output:
```

Comment:
I think this `Check_Output` dataclass is only for the `csv` mode, right?

Reply:
Right now yes